File: VB/openai login design/UserControlLogin.vb
Public Class UserControlLogin
    Private Sub LinkLabel1_LinkClicked(sender As Object, e As LinkLabelLinkClickedEventArgs) Handles LinkLabel1.LinkClicked
        Form1.Guna2Panel1.Controls.Clear()
        Form1.Guna2Panel1.Controls.Add(New UserControlSignUp())
    End Sub
End Class

File: C#/openai login design/UserControlLogin.cs
using System.Windows.Forms;

namespace openai_login_design
{
    public partial class UserControlLogin
    {
        public UserControlLogin()
        {
            InitializeComponent();
        }
        private void LinkLabel1_LinkClicked(object sender, LinkLabelLinkClickedEventArgs e)
        {
            My.MyProject.Forms.Form1.Guna2Panel1.Controls.Clear();
            My.MyProject.Forms.Form1.Guna2Panel1.Controls.Add(new UserControlSignUp());
        }
    }
}